### DISCUSSION TITLE
GoPro Hero 10 Bones Shutting Down Mid Recording

### DISCUSSION THREAD

**Discussion by goprohero10bones** [gopro/labs#1007] [Q&A]
There are multiple Hero 10 Bones which sporadically seem to turn off while recording between 30 to 200 seconds of recording. Sending a trigger pulse seems to turn them back on and they start recording. But when power cycled, they lose their archived settings. 

I think they could be overheating but even with a fan blowing on them, they'll still shut down. The QR setting is :!EmVdVq1mVr4p120fW0v0q0dVoR1CW0oT240722115436!MWAKE=1!MARCH=1"Archive Mode\nEnabled\nShutting-down"!O 

I'm pretty new to this, please let me know if there are settings to reduce heating/power consumption or if there's a setting I can change to stop them from randomly turning off. Thanks!

**Comment by dnewman-gpsw** [comment]
Don't use Archive mode on Bones, it was never tested with that hardware (it is not list as supporting Bones.)

Start with no Labs hacks.  And select a simple mode like 4k @ 30gps : command mVr4p30

Does that work?  With got airflow will that record continuously? 

If that works try mVr4p60, and so on.

You are asking for 4K @ 120, that is a hard mode for all cameras, and hardest for a caseless camera, so it will need an excellent airflow.

Then try mVr4p120
